submissions/xT0PERx/js-dom/code.js

```diff
@@ -0,0 +1,65 @@
+  },
+];
+
+document.querySelector('.menu-list').addEventListener('click', function (e) {
+  if (e.target && e.target.nodeName === 'LI') {
+    switch (e.target.className) {
+      case 'menu-item-1':
+        showContent('menu-item-1');
```

Comment:
You need to come up with an option, get the card index and transfer it to showContent, then the code will not be duplicated

Reply:
Could you explain in more detail, I don't quite understand what you mean.